outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "network_id" {
```

Comment:
add outputs to root module to display created network, subnetwork, and firewall ids, details of created VMs, and URLs of storage buckets